src/ui/server/helpers/flatten-application-data/index.ts

```diff
@@ -1,5 +1,28 @@
 import getTrueAndFalseAnswers from '../get-true-and-false-answers';
-import { Application, ApplicationFlat } from '../../../types';
+import { Application, ApplicationFlat, ApplicationPolicyContact } from '../../../types';
+import INSURANCE_FIELD_IDS from '../../constants/field-ids/insurance';
+
+const {
+  POLICY_AND_EXPORTS: {
+    NAME_ON_POLICY: { IS_SAME_AS_OWNER, POSITION, POLICY_CONTACT_EMAIL },
+  },
+  ACCOUNT: { FIRST_NAME, LAST_NAME, EMAIL },
+} = INSURANCE_FIELD_IDS;
+
+/**
+ * policyContactMapped
+ * maps policyContact and replaces email with POLICY_CONTACT_EMAIL for task list
+ * @param policyContact
+ * @returns {Object}
+ */
+export const policyContactMapped = (policyContact: ApplicationPolicyContact) => ({
```

Comment:
`mapPolicyContact`?

Also can we add a note about `POLICY_CONTACT_EMAIL` please RE required dot notation and flattened application data?